seed/tests/test_meter_views.py

```diff
@@ -24,15 +28,120 @@ def setUp(self):
         self.client.login(**self.user_details)
 
     def test_view_that_returns_valid_types_and_units_for_meters(self):
-        url = reverse('api:v3:meters-valid-types-units')
+        url = reverse('api:v3:properties-valid-meter-types-and-units')
 
         result = self.client.get(url)
         result_dict = ast.literal_eval(result.content.decode("utf-8"))
 
         expectation = {
-            type: list(units.keys())
-            for type, units
-            in kbtu_thermal_conversion_factors("US").items()
+            type: list(units.keys()) for type, units in conversion_factors("US").items()
         }
 
         self.assertEqual(result_dict, expectation)
+
+
+class TestMeterCRUD(DataMappingBaseTestCase):
+    def setUp(self):
+        self.user_details = {
+            'username': 'test_user@demo.com',
+            'password': 'test_pass',
+        }
+        self.user = User.objects.create_superuser(
+            email='test_user@demo.com', **self.user_details
+        )
+        self.org, _, _ = create_organization(self.user)
+        self.client.login(**self.user_details)
+
+        # faker class for properties
+        self.property_view_factory = FakePropertyViewFactory(organization=self.org)
+
+    def test_create_meter(self):
+        property_view = self.property_view_factory.get_property_view()
+        url = reverse('api:v3:property-meters-list', kwargs={'property_pk': property_view.id})
+
+        payload = {
+            'property_id': property_view.property.id,
```

Comment:
I'm pretty sure this is correct, but a second set of eyes will be nice! 

The thought is that we find the "property" using the property_view_id that is in the URL (which is the `reverse` command in line 60, then the meter is attached to the "property" ID which is found through the property_view.

Reply:
I'm a little confused. The request is `POST /api/v3/property_meters_list/<property_view.id>`, then in the body we give it the `property_id`. But like, why would the body need to specify the property, as it is a given because of the `property_view.id` in the url? 


The request being made here looks right, the actual endpoint seems a bit repetitive.

Reply:
I updated this so that the meter's viewset `perform_create` method grabs the `property_pk`. The body had the property because that was legacy, it has now been removed.